app/models/measurement.rb

```diff
@@ -1,5 +1,7 @@
 class Measurement < ActiveRecord::Base
   belongs_to :person
+  scope :descending, -> { order(date: :desc) }
+  scope :latest, -> { order(date: :asc).last }
```

Comment:
AHA! THAT explains a lot! (Esp. about the Teams app.)
One question though: I tried another syntax: 

```ruby
{ order(date: :asc).limit(1) }  
```
What is wrong with this?